### PR TITLE
Added Qubino Flush Shutter

### PR DESCRIPTION
**Pull request by SkynetHome**
Added ZMNHCD per Issue #2651

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/qubino/zmnhcd.xml [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+				<Value>0</Value>
+				<Label lang="en">0 - ALL ON is not active ALL OFF is not active
+				</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">1 - ALL ON is not active ALL OFF active</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/qubino/zmnhcd.xml]: Value 1 appears twice

File: bundles/binding/org.openhab.binding.zwave/database/qubino/zmnhcd.xml [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+	</Configuration>
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>1</Maximum>
+			<Label lang="en">Controller Updates</Label>
+			<Help lang="en"> default reporting group (reserved for the main
+				controller)
+			</Help>
+		</Group>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/qubino/zmnhcd.xml]: Group 1 probably should have ```<SetToController>true</SetToController>```

**Comment by SkynetHome** [comment]
Fixing the SetToController but my copy doesnt show Value 1 twice its correct with values of 255 0 1 2? not sure how you could be seeing something different?

**Comment by SkynetHome** [comment]
Never mind was in wrong file making changes and will try to commit properly!

**Comment by SkynetHome** [comment]
Here ya go Chris as with the other PR I think I FINALLY got down how to Squash my commits!

**Comment by cdjackson** [comment]
Thanks (again :) )

**Comment by cdjackson** [comment]
Ah - no - we have a merge conflict...

Do you fancy trying to resolve this, or would you prefer me to?

**Comment by SkynetHome** [comment]
Ahhh, thats what I get for congratulating myself. I will be on the road today for 6  hours heading out of town with my wife for a work engagement of hers. If you have time to sort it out before then, great if not I should have plenty of free time once we get to the resort to work on it.

**Comment by SkynetHome** [comment]
Just checked source tree and git and I have no files showing conflicts. Not sure what is causing this. Only thing that is of lingering concern is that when I fetch from all remotes and then pull openhab/master into my current branch my repository on github still says its 10 commits behind and yet when I compare its says there is nothing to compare as both branches are even. This has been an ongoing state and has not affected other PR's. I cant find anything on my end that is causing conflict. That being said should I also being pulling  remote origin/master into current branch after fetch or just openhab/master?

**Comment by cdjackson** [comment]
You won’t see any files with conflicts - the conflicts will only occur when you merge master into the branch…

**Pull request closed without merging**